api_tests/package.json

```diff
@@ -7,14 +7,33 @@
         "check": "tsc && prettier --check '**/*.{ts,json,yml}' && tslint --project .",
         "postinstall": "mkdir -p ./gen && json2ts -i ./siren.schema.json -o ./gen/siren.d.ts",
         "pretest": "cargo build --bin cnd",
-        "test": "ts-node ./harness.ts",
+        "test": "tsc && jest --detectOpenHandles",
+        "tsc": "tsc",
         "fix": "tslint --project . --fix && prettier --write '**/*.{ts,js,json,yml}'"
     },
     "engines": {
         "node": "^10.14"
     },
     "author": "CoBloX Team",
     "license": "ISC",
+    "jest": {
+        "preset": "ts-jest",
+        "roots": [
+            "./tests"
+        ],
+        "testRegex": "\\.ts$",
+        "transform": {
+            "^.+\\.(t|j)s$": "ts-jest"
+        },
+        "moduleFileExtensions": [
+            "ts",
+            "js",
+            "json",
+            "node"
+        ],
+        "testEnvironment": "./dist/src/test_environment",
+        "testTimeout": 180000
```

Comment:
I see you moved the timeout.
Having the timeout within the `nActorTest` function was actually on purpose so we can dump state before failing.

Reply:
Ok. Didn't know this was on purpose. I'll reaadd it.